package.json

```diff
@@ -12,8 +12,9 @@
     "eslint": "^7.8.1",
     "gulp": "^4.0.2",
     "gulp-if": "^3.0.0",
-    "gulp-sass": "^4.1.0",
-    "merge-stream": "^2.0.0"
+    "gulp-sass": "^5.0.0",
+    "merge-stream": "^2.0.0",
+    "sass": "1.32.13"
```

Comment:
Note that `sass` is 🔒 locked into `1.32.13`. Otherwise every single SCSS compile gives 30+ warnings about the Protocol library that [cannot be quieted](https://github.com/sass/libsass/issues/2822#issuecomment-482914373). It makes the console output impossible.